src/main/java/org/broadinstitute/hellbender/utils/bigquery/TableReference.java

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.hellbender.utils.bigquery;
+
+import java.util.List;
+
+public class TableReference {
+    public final String tableProject;
+    public final String tableDataset;
+    public final String tableName;
+    public final List<String> fields;
+
+    public TableReference(String fullyQualifiedTableName, List<String> fields) {
+        String[] vals = fullyQualifiedTableName.split("\\.");
+        tableProject = vals[0];
+        tableDataset = vals[1];
+        tableName = vals[2];
```

Comment:
Handle the case where there aren't enough tokens after splitting by throwing an `IllegalArgumentException`. Also check for empty tokens.